ui-supplemental/css/custom.css

```diff
@@ -88,7 +126,7 @@ kbd {
   font-family: "Roboto Mono", monospace;
 }
 
-.manpage .doc > h1.page:first-child {
+.manpage .doc>h1.page:first-child {
```

Comment:
@divyanshugarg36 Is there any specific reason to remove space from here?

Reply:
This happened because of my formatter which is configured with style lint, I can revert this if you suggest

Reply:
Yeah sure, I think the spacer one looks clear and more breathable.